docs/asciidoc/modules/ROOT/pages/overview/apoc.trigger/index.adoc

```diff
@@ -46,6 +71,16 @@ CALL apoc.trigger.resume(name) \| it resumes the paused trigger
 |xref::overview/apoc.trigger/apoc.trigger.propertiesByKey.adoc[apoc.trigger.propertiesByKey icon:book[]]
 
 
+|label:function[]
+|label:apoc-full[]
+|xref::overview/apoc.trigger/apoc.trigger.toNode.adoc[apoc.trigger.toNode icon:book[]]
+
+
+|label:function[]
+|label:apoc-full[]
+|xref::overview/apoc.trigger/apoc.trigger.toRelationship.adoc[apoc.trigger.toRelationship icon:book[]]
+
+
 |label:function[]
 |label:apoc-full[]
 |===
```

Comment:
To make this page clearer, I was wondering if we couldn't split the page into two sections? The top section would contain the new trigger procedures with a brief explanation of the new changes (eg they are on a per-database basis and work in a cluster environment). The bottom section would contain the old trigger procedures with a brief mention that they are now deprecated.

I think this would help users view at a glance what they should and shouldn't be focusing on.

What do you think?

Reply:
Yes, good idea. I will add it.

Reply:
Indeed this `xref` links point to `background-operations/triggers.adoc`, so I added the explanation directly on that page, as the user does not necessarily start from this page when looking for the documentation